data/data-pipeline/data_pipeline/etl/sources/census_acs/etl.py

```diff
@@ -55,59 +64,136 @@ def __init__(self):
             "Median value ($) of owner-occupied housing units"
         )
 
+        # Educational attainment figures
+        self.EDUCATION_POPULATION_OVER_25 = "B15003_001E"  # Estimate!!Total
+        self.EDUCATION_NO_SCHOOLING = (
+            "B15003_002E"  # Estimate!!Total!!No schooling completed
+        )
+        self.EDUCATION_NURSERY = (
+            "B15003_003E"  # Estimate!!Total!!Nursery school
+        )
+        self.EDUCATION_KINDERGARTEN = (
+            "B15003_004E"  # Estimate!!Total!!Kindergarten
+        )
+        self.EDUCATION_FIRST = "B15003_005E"  # Estimate!!Total!!1st grade
+        self.EDUCATION_SECOND = "B15003_006E"  # Estimate!!Total!!2nd grade
+        self.EDUCATION_THIRD = "B15003_007E"  # Estimate!!Total!!3rd grade
+        self.EDUCATION_FOURTH = "B15003_008E"  # Estimate!!Total!!4th grade
+        self.EDUCATION_FIFTH = "B15003_009E"  # Estimate!!Total!!5th grade
+        self.EDUCATION_SIXTH = "B15003_010E"  # Estimate!!Total!!6th grade
+        self.EDUCATION_SEVENTH = "B15003_011E"  # Estimate!!Total!!7th grade
+        self.EDUCATION_EIGHTH = "B15003_012E"  # Estimate!!Total!!8th grade
+        self.EDUCATION_NINTH = "B15003_013E"  # Estimate!!Total!!9th grade
+        self.EDUCATION_TENTH = "B15003_014E"  # Estimate!!Total!!10th grade
+        self.EDUCATION_ELEVENTH = "B15003_015E"  # Estimate!!Total!!11th grade
+        self.EDUCATION_TWELFTH_NO_DIPLOMA = (
+            "B15003_016E"  # Estimate!!Total!!12th grade, no diploma
+        )
+
+        self.EDUCATIONAL_FIELDS = [
+            self.EDUCATION_POPULATION_OVER_25,
+            self.EDUCATION_NO_SCHOOLING,
+            self.EDUCATION_NURSERY,
+            self.EDUCATION_KINDERGARTEN,
+            self.EDUCATION_FIRST,
+            self.EDUCATION_SECOND,
+            self.EDUCATION_THIRD,
+            self.EDUCATION_FOURTH,
+            self.EDUCATION_FIFTH,
+            self.EDUCATION_SIXTH,
+            self.EDUCATION_SEVENTH,
+            self.EDUCATION_EIGHTH,
+            self.EDUCATION_NINTH,
+            self.EDUCATION_TENTH,
+            self.EDUCATION_ELEVENTH,
+            self.EDUCATION_TWELFTH_NO_DIPLOMA,
+        ]
+
+        self.HIGH_SCHOOL_ED_RAW_COUNT_FIELD = (
+            "Individuals age 25 or over with less than high school degree"
+        )
+        self.HIGH_SCHOOL_ED_FIELD = "Percent individuals age 25 or over with less than high school degree"
+
         self.STATE_GEOID_FIELD_NAME = "GEOID2"
+
         self.df: pd.DataFrame
 
+    @classmethod
```

Comment:
These are class methods now so they can be reused a little easier in the other Census ACS 2010 ETL.

Reply:
I have the concept of `etl_utils.py` on the cebsus etl, the idea being that other etls can include them and we flag those classes/functions as importable in other places.

Reply:
Yeah true. @esfoobar-usds should i just take both these class methods and move them to `etl_utils`?

Reply:
Based on your suggested I think that would be cleaner so I would vote yes.

Reply:
To be super clear, it would be a new file at `data/data-pipeline/data_pipeline/etl/sources/census_acs/etl_utils.py`

Reply:
Oh I see!

Reply:
Then from there, both `acs`. and `acs_2010` can import from there.